ApeWorX/ape
# fix: add argument conversion to contract estimate gas API

…ct instance

### What I did



fixes: #931 

### How I did it

The same convert_tuple strategy we use elsewhere when interacting with the `ContractInstance`

### How to verify it

See tests!

Make sure you can pass accounts, contract instances, magic values like "100 gwei" etc to the estimate gas RPCs

### Checklist
